OpenSim/Common/Test/testPath.cpp

```diff
@@ -0,0 +1,125 @@
+/* ------------------------------------------------------------------------- *
+*                        OpenSim:  testPath.cpp                              *
+* -------------------------------------------------------------------------- *
+* The OpenSim API is a toolkit for musculoskeletal modeling and simulation.  *
+* See http://opensim.stanford.edu and the NOTICE file for more information.  *
+* OpenSim is developed at Stanford University and supported by the US        *
+* National Institutes of Health (U54 GM072970, R24 HD065690) and by DARPA    *
+* through the Warrior Web program.                                           *
+*                                                                            *
+* Copyright (c) 2005-2016 Stanford University and the Authors                *
+* Author(s): Carmichael Ong                                                  *
+*                                                                            *
+* Licensed under the Apache License, Version 2.0 (the "License"); you may    *
+* not use this file except in compliance with the License. You may obtain a  *
+* copy of the License at http://www.apache.org/licenses/LICENSE-2.0.         *
+*                                                                            *
+* Unless required by applicable law or agreed to in writing, software        *
+* distributed under the License is distributed on an "AS IS" BASIS,          *
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   *
+* See the License for the specific language governing permissions and        *
+* limitations under the License.                                             *
+* -------------------------------------------------------------------------- */
+
+#include <OpenSim/Common/ComponentPath.h>
+#include <OpenSim/Auxiliary/auxiliaryTestFunctions.h>
+
+/* The purpose of this test is strictly to check that the Path functions work
+ * outside of the context of the objects/components they are meant to service. 
+ */
+
+using namespace OpenSim;
+using namespace std;
+
+void testComponentPath() {
+    // Test that input matches printing out
+    string absStr1 = "/a/b/c/d";
```

Comment:
To help future developers with preserving the integrity of these tests, can you add comments explaining what each of these paths are intended to test?